plugin/main.cpp

```diff
@@ -19,7 +19,8 @@
 Q_DECL_EXPORT int main(int argc, char *argv[])
 {
     QLoggingCategory::setFilterRules(QStringLiteral("geoclue.provider.mlsdb.debug=false\n"
-                                                    "geoclue.provider.mlsdb.position.debug=false"));
+                                                    "geoclue.provider.mlsdb.position.debug=false\n"
+                                                    "geoclue.provider.mlsdb.online.debug=false"));
```

Comment:
I thought that they should be false by default, however when I removed these filter rules on my device they all got output.  I don't know why.  Perhaps there is some specific semantics for category names which get wildcard matched (e.g., only qt*debug are suppressed by default?)